extensions/cornerstone-3d/src/Viewport/OHIFCornerstone3DViewport.tsx

```diff
@@ -65,6 +73,9 @@ const OHIFCornerstoneViewport = React.memo(props => {
     // of the imageData in the OHIFCornerstone3DViewport. This prop is used
     // to set the initial state of the viewport's first image to render
     initialImageIdOrIndex,
+    // CINE props.
+    CINEIsPlaying,
```

Comment:
Isn't this bad to have in props for a viewport as it will probably trigger re-renders all the time?

Shouldn't we expose this through a utility service and do it by ID instead or something?